include/math_utils.h

```diff
@@ -0,0 +1,174 @@
+/*
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Copyright (C) 2020-2022  The DOSBox Staging Team
+ *  Copyright (C) 2002-2021  The DOSBox Team
```

Comment:
Nice organisation cleanup!
I believe all of these math routines were added by the Staging team (exclusively).

Reply:
Updated the header.